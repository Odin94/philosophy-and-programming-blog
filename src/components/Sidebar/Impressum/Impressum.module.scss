@use "sass:color";

@import "../../../assets/scss/variables";
@import "../../../assets/scss/mixins";

.impressum {
  color: color.adjust($color-gray, $whiteness: 18%);
  font-size: $typographic-small-font-size;
}