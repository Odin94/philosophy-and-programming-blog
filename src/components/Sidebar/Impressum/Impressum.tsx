import React from "react";

import * as styles from "./Impressum.module.scss";

const Impressum = () => (
  <a className={styles.impressum} href="/pages/impressum">Impressum</a>
);

export default Impressum;
