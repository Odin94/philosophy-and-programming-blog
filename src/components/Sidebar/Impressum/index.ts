export { default as Impressum } from "./Impressum";
