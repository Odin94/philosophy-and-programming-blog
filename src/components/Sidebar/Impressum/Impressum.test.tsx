import React from "react";
import renderer from "react-test-renderer";

import { Impressum } from "@/components/Sidebar/Impressum";

describe("Impressum", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Impressum />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
