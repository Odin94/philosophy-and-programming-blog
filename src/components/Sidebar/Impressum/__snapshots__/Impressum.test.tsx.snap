// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Impressum renders correctly 1`] = `
<a
  href="/pages/impressum"
>
  Impressum
</a>
`;
